package.json

```diff
@@ -1,16 +1,16 @@
 {
   "license": "Unlicensed",
   "private": true,
-  "repository": {
-  },
+  "repository": {},
   "dependencies": {
     "babel-brunch": "~6.0.0",
+    "babel-preset-es2015": "^6.9.0",
```

Comment:
It's used on Babel Brunch for transpile Javascript files, when it's not present Phoenix throws an exception.

```
19 Jul 01:39:44 - error: Compiling of web\static\js\app.js failed. Couldn't find preset ".\\node_modules\\babel-preset-es2015" relative to directory "web\\static\\js" ; Compiling of web\static\js\socket.js failed. Couldn't find preset ".\\node_modules\\babel-preset-es2015" relative to directory "web\\static\\js"
```